templates/takeovers/_german_ai_webinar.html

```diff
@@ -0,0 +1,162 @@
+<section lang="de" class="p-strip is-deep p-takeover--ai-webinar js-takeover">
+  <div class="row u-equal-height">
+    <div class="col-6">
+      <h1 class="p-takeover__title">Alles, was Sie über Ubuntu, Sicherheit und Compliance wissen müssen</h1>
+      <p class="p-takeover__text">Erfahren Sie mehr über Ubuntu, eine führende Technologie für regulatorische Anforderungen in verschiedensten Branchen.</p>
+      <div class="u-hide--small">
+        <a href="https://www.brighttalk.com/webcast/17086/351767?utm_source=Takeover&utm_medium=Takeover&utm_campaign=CY19_DC_DACH_OpenStack_WBN_ComplianceDE_takeover" class="p-button--positive u-no-margin--bottom"><span class="p-link--external">Webinar ansehen</span></a>
```

Comment:
```suggestion
        <a href="https://www.brighttalk.com/webcast/17086/352220?utm_source=takeover&utm_medium=takeover&utm_campaign=CY19_DC_DACH_Kubernetes_WBN_AIMLDE_Takeover " class="p-button--positive u-no-margin--bottom"><span class="p-link--external">Webinar ansehen</span></a>
```

The copy doc is different